hugegraph-server/hugegraph-dist/src/assembly/static/bin/dump-store.sh

```
@@ -30,22 +30,26 @@ TOP="$(cd $BIN/../ && pwd)"
-conf=$1
 if [ $# -eq 0 ]; then
-    conf=$CONF/hugegraph.properties
+    set -- "$CONF/graphs/hugegraph.properties"
 fi
+conf=$1
 
 cd $TOP
 
 echo "Dumping HugeGraph Store($conf)..."
 
 dump_store_ext_jar_path=$LIB/hugegraph-dist-*.jar
```

Shellcheck (0.11.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)